pkg/roachprod/vm/aws/aws.go

```diff
@@ -66,8 +66,14 @@ func Init() error {
 		"(https://docs.aws.amazon.com/cli/latest/userguide/installing.html)"
 	const noCredentials = "missing AWS credentials, expected ~/.aws/credentials file or AWS_ACCESS_KEY_ID env var"
 
-	providerInstance := &Provider{}
-	providerInstance.Config.awsConfig = *DefaultConfig
+	providerInstance, err := NewProvider()
+	if err != nil {
+		vm.Providers[ProviderName] = flagstub.New(
+			&Provider{},
+			fmt.Sprintf("unable to init aws provider: %s", err),
+		)
+		return errors.New("doesn't have the required version")
```

Comment:
Shouldn't we return the original error here? 
e.g. `return errors.Wrap(err, "unable to init aws provider")`

Reply:
Sorry, quick copy/paste that went wrong. Fixing.